CMakeLists.txt

```diff
@@ -3,7 +3,24 @@ project(openGA CXX C)
 set(CMAKE_CXX_STANDARD 11)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
-find_package(Boost COMPONENTS system filesystem thread chrono REQUIRED)
+# Eigen
+include_directories("/usr/include/eigen3" )
```

Comment:
Does this path work for a cross platform application?

Reply:
The OpenMP is for two GA running parallel. It is easier than `std::thread`. This modification is just for me.
CMake is for somebody that has no experience with Make. It should not influence the Make.